src/components/AddItem.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { useState } from 'react';
+
+export default function AddItem({ handleAddItem  }) {
+    const [text, setText] = useState('');
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        handleAddItem(text);
```

Comment:
adding setText('') makes for a nice user experience to clear out the input after the item has been added